### ISSUE TITLE
Error when switching profile on VScodium

### ISSUE THREAD

**Issue by BenoitGeslain** [zokugun/vscode-sync-settings#44]
Upon switching profile, the extension fails to restart VSCodium and throws this error:
`Sync Settings: Error: Can determine binary path`

Version: 1.76.2
Release: 23074
Commit: 22a808587994126e19125c6a1a03c00b990223f4
Date: 2023-03-15T18:20:11.806Z
Electron: 19.1.9
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Linux x64 6.2.8-arch1-1
Sandboxed: No

Name: Sync Settings
Id: zokugun.sync-settings
Description: Easily synchronize your settings
Version: 0.16.0
Publisher: zokugun
VS Marketplace Link: https://open-vsx.org/vscode/item?itemName=zokugun.sync-settings

**Comment by daiyam** [comment]
Which distro? How was VSCodium was installed?

**Comment by BenoitGeslain** [comment]
EndeavourOS up to date, I installed VSCodium using yay (aur/vscodium-bin)

**Comment by daiyam** [comment]
No, but it's used only to automatically restart the app so you can do it manually...

**Comment by lifedanceman** [comment]
Do yum plan to do integration with native profiles?
